ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/loader/JsonTestLoader.java

```diff
@@ -67,9 +67,12 @@ public Stream<T> load() {
         ? loadTestPathsFromDirectory()
         : getTestPathsFromWhiteList(whiteList);
 
+    final String testRegex = System.getProperty("ksql.functional.test.regex");
```

Comment:
Small side-change I made to help debug some tests. This replaces the RQTT-specific test filter I'd added before.